lib/cfme/cloud_services/inventory_sync.rb

```diff
@@ -29,6 +29,27 @@ def self.sync_queue(userid, targets)
         MiqTask.generic_action_with_callback(task_opts, queue_opts)
       end
 
+      def self.collect(opts)
```

Comment:
TODO: need a better name than `.collect`

Reply:
oh yeah...as discussed, .collect overloads with the core method and is confusing.

Reply:
ditto for naming update, as we'd probably expose the same on the api side of things.

Reply:
Okay renamed this to `.bundle`